src/index.css

```diff
@@ -0,0 +1,185 @@
+/* green: #677767 */
+/* beige: #F7F6F1 */
+/* white: #fff */
+/* black: #000 */
+
+html, body {
+    height: 100%;
+    width: 100%;
+    overflow-x: hidden;
+    font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    font-weight: 300;
+
+    min-width: 320px;
+    line-height: 1.2;
+    background-color: #F7F6F1;
+}
+
+body,
+#dashboard,
+.dashboard__heading,
+.dashboard__card-heading {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#authorization {
+    display: none;
+}
+
+
+/* Authorization */
+
+
+form {
+    width: 100%;
+    padding: 6.5% 12%;
+    text-align: center;
+}
+
+form, .dashboard__card {
+    background-color: #fff;
+    border-radius: 2px;
+    box-shadow: rgba(149, 157, 165, 0.2) 0 8px 24px;
+}
+
+form .inputs input {
+    margin: 7% 0;
+    width: 100%;
+    line-height: 1.5;
+
+    border: none;
+    border-bottom: 0.6px solid rgba(0, 0, 0, 0.2);
+}
+
+button {
+    cursor: pointer;
+
+    font-size: 1.1em;
+    font-weight: 500;
+
+    margin: 4% 0;
+    padding: 1.7% 7%;
+
+    color: #677767;
+    background-color: transparent;
+
+    border-radius: 2px;
+    border: 2px solid #677767;
+
+    transition: .5s;
+}
+
+button:hover,
+button:active,
+button:focus {
+    box-shadow: none;
+    outline: none;
+
+    color: #fff;
+    background-color: #677767;
+}
+
+form .inputs input::placeholder {
+    opacity: 0.5;
+}
+
+form .inputs input:focus {
+    outline: none;
+}
+
+form > p {
+    font-size: 0.9em;
+}
+
+form > p > span {
+    font-weight: 500;
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+form > p > span:hover {
+    color: #677767;
+    text-decoration: underline;
+}
+
+
+/*Dashboard */
+
+
+#dashboard {
+    width: 100%;
+    height: 100%;
+
+    padding: 2% 5%;
+
+    align-items: flex-start;
+}
```

Comment:
If the user has more than 4 statuses, this block should scroll horizontally.

Reply:
Fixed by making table width fixed size.